templates/type_conversions

```diff
@@ -0,0 +1,12 @@
+{
+    "double": "Double",
```

Comment:
This tells the generator how to convert a C++ type to a Rust type. Note that this has to correspond to what protobuf does. I usually take it from [here](https://developers.google.com/protocol-buffers/docs/proto3#scalar), but Rust is not officially documented, apparently.